src/app/ApplicationTemplate.UWP/Views/Content/Posts/EditPostPage.xaml

```diff
@@ -15,10 +14,14 @@
 			<RowDefinition Height="*" />
 		</Grid.RowDefinitions>
 
+		<!-- CommandBar -->
 		<CommandBar Content="{Binding Title}">
 			<toolkit:CommandBarExtensions.NavigationCommand>
-				<AppBarButton Command="{Binding Cancel}">
+				<AppBarButton Command="{Binding Cancel}"
+							  Foreground="{StaticResource MaterialSurfaceBrush}">
```

Comment:
Surface is a background, not a foreground.

Reply:
Actually since the color the background is 001E60 OnBackground (Could also be OnSecondary or OnSurface) it would make more sense if all the instance of white were MaterialBackground since

![image](https://user-images.githubusercontent.com/64613932/177182589-8034d84d-9eb5-46ad-92e3-e76e5a934b0f.png)